HW11/task1.py

```diff
@@ -0,0 +1,75 @@
+phonebook = {}
+
+def stats():
+    print("Number of records:", len(phonebook))
+
+def add():
+    name = input("Input name: ")
+    try:
+        if name in phonebook:
+            print("Error: entry with name", name, "already exists")
+        else:
+            phone = input("Enter a phone number: ")
+            lastname = input("Enter Lastname: ")
+            phonebook[name] = {'Phone number': phone, 'Lastname': lastname}
+        print("The entry is added to the phone book")
+    except Exception as e:
+        print("Error:", str(e))
+
+def delete(name):
+    try:
+        if name in phonebook:
+            del phonebook[name]
+            print("Entry by name", name, "deleted from phone book")
+        else:
+            print("Error: record with name", name, "does not exist")
+    except Exception as e:
+        print("Error:", str(e))
```

Comment:
real exception should be handled there, it could be like:
```suggestion
def delete(name):
    try:
        del phonebook[name]
        print("Entry by name", name, "deleted from phone book")
    except KeyError as e:
        print(f"Error: record with name {name} does not exist. Exc: {e}")
```